ddpui/tests/core/test_dbtfunctions.py

```
@@ -78,7 +80,39 @@ def test_map_airbyte_destination_spec_to_dbtcli_profile_success_ssl_params(tmpdi
+    res = map_airbyte_destination_spec_to_dbtcli_profile(conn_info, dbt_project_params)
+    assert res["sslmode"] == "require"
+    assert "sslrootcert" not in res
+    assert "sslrootcert_content" not in res
+
+
+def test_map_airbyte_destination_spec_to_dbtcli_profile_ssl_no_org_project_dir():
+    """Tests ssl with ca_certificate but no dbt_project_params raises"""
+    conn_info = {"ssl_mode": {"mode": "verify-ca", "ca_certificate": "ca_certificate"}}
+    try:
+        map_airbyte_destination_spec_to_dbtcli_profile(conn_info, None)
+        assert False, "should have raised"
```

Ruff (0.15.12)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)